### PR TITLE
Update musicadmin.html

### PR DESCRIPTION
**Pull request by PaulAntonDeen**
Add delete button to cards

### CODE REVIEW

File: musicadmin.html [hunk 1]
```diff
@@ -133,6 +134,15 @@ <h6 v-if="card.title" class="card-title">{{ card.title }}</h6>
 							this.saving = false;
 						});
 					},
+					deleteCard (item) {
+					    axios
```
Comment (tader) [inline musicadmin.html:139]: Add: `this.saving = true;`

File: musicadmin.html [hunk 2]
```diff
@@ -133,6 +134,15 @@ <h6 v-if="card.title" class="card-title">{{ card.title }}</h6>
...
+						.get('', {
+							params: {
+								drop: this.tag_id
+							}
+						})
+						this.editing = false;
+					}
```
Comment (tader) [inline musicadmin.html:149]: Add: `this.saving = false;`

below editing = false;

File: musicadmin.html [hunk 3]
```diff
@@ -133,6 +134,15 @@ <h6 v-if="card.title" class="card-title">{{ card.title }}</h6>
...
 						});
 					},
+					deleteCard (item) {
+					    axios
+						.get('', {
+							params: {
+								drop: this.tag_id
+							}
+						})
+						this.editing = false;
```
Comment (tader) [inline musicadmin.html]: this.editing = false; (and saving) MUST go in a `.then((response) => { /* here */ });`

**Review by tader — CHANGES_REQUESTED** [review]
Als the item should be removed from the local state… I’ll look int that.

**Comment by PaulAntonDeen** [comment]
Done

**Pull request merged**